nodejs/docs/node-standard-and-best-practices.md

```diff
@@ -100,22 +100,22 @@ To keep consistency between all our projects we define a convention for a sort o
 
 ### 3.1- Files
 
-File names must be **snake_case** and **plural** except models, which will be in **singular**. We like to keep models singular, since they are representing a singular instance of that type of objects, but the other type of files in plural since they usually handle collections of objects. Also we like to avoid the entity (model, controller, etc) of the file in the name itself.
+File names must be **snake_case** and **plural**, with the exception of models, which will be in **singular**. We like to keep models singular, since they are representing a singular instance of that type of object, but the other type of files should be kept in plural since they usually handle collections of objects. Also we like to avoid adding the entity (model, controller, etc) of the file in the name itself.
 For example:
-- if we have the model user, the file should be named **user.js** instead of **users.js** **user.model.js** or **user_model.js**.
+- if we have the model user, the file should be named **user.js** instead of **users.js**, **user.model.js** or **user_model.js**.
 - if we have the cars controller, the file should be named **cars.js** instead of **car.js**, **cars.controller.js** or **cars_controller.js**
 
 ### 3.2- Routes, input and output API parameters
 
-With the purpose of unifying the interfaces with other techs and making things easier to the client who will consume other API, we decide to keep routes, input and output API parameters in **snake_case**.
+With the purpose of unifying the interfaces with other techs and making things easier to the client who may consume any other API, we decided to keep routes, input and output API parameters in **snake_case**.
```

Comment:
With the purpose of _unify_ interfaces between techs and make things easier to clients who may consume any other API, we decided to keep routes, input and output API parameters in **snake_case**.

Reply:
In this context, the use of the present continuous form of `unify` is correct. To use it's infinitive form, we should rephrase this, maybe write something like `In order to unify the interfaces...`. Both options seem equally valid to me, what do you think?